synapse/models/orgs.py

```diff
@@ -162,6 +162,9 @@
                 'doc': 'A position within an org which can be organized into an org chart with replacable contacts.'}),
 
             ('ou:attendee', ('guid', {}), {
+                'interfaces': (
+                    ('entity:contactable', {'template': {'contactable': 'attendee'}}),
```

Comment:
Per discussion should also inherit entity:abstract to allow for attendee entity resolution.